### PR TITLE
[W5.11][W12-1]Elston Aw

### PR DESCRIPTION
**Pull request by elstonayx**
Ready for review.

### CODE REVIEW

**Comment by bennchong** [comment]
Made commands into non-case sensitive. Really helpful when commands are entered quickly. +1 for being a beast climber

**Review by jitwei98 — CHANGES_REQUESTED** [review]
Smart enhancement to have here.
Would like to see some test to know if it works or not.

File: src/seedu/addressbook/parser/Parser.java [hunk 1]
```diff
@@ -70,7 +70,7 @@ public Command parseCommand(String userInput) {
             return new IncorrectCommand(String.format(MESSAGE_INVALID_COMMAND_FORMAT, HelpCommand.MESSAGE_USAGE));
         }
 
-        final String commandWord = matcher.group("commandWord");
+        final String commandWord = matcher.group("commandWord").toLowerCase();
```
Comment (sylchw) [inline src/seedu/addressbook/parser/Parser.java:73]: good for convenience, but do edit the test cases

**Review by sylchw — APPROVED** [review]
Good change for convenience

**Review by okkhoy — APPROVED** [review]
This enhancement surely gives you a tick mark on the graded LO.
However, not sure if the intended objective of being able to work with an intermediate size codebase was achieved.

**Comment by okkhoy** [comment]
@elstonayx 
Some comments added. Please take note and close the pull request.

**Pull request closed without merging**